doc/contribution-guidelines.md

```diff
@@ -0,0 +1,97 @@
+# Contribution guidelines
+
+- [Pull Requests](#pullrequest)
+- [Commit Message Guidelines](#commit)
+
+## <a name="pullrequest"></a> Pull Requests Guidelines
+Pull requests are the only means by which you can contribute to this project, please follow the following steps when submitting pull requests :
+
+1. Create your pull request. (PR name must match [commit name](#commit))
+2. Fill the pull request template with all of the required elements
+3. CI tests kick in and report back any issues, you should fix these issues before continuing. No broken PR will be reviewed.
+4. The reviewer(s) uses Github review system so you will know if the reviewer requested any changes, approved the Pull Request or simply added comments.
+5. If any changes are requested please fix them and then once you are ready ask for a new review
+
+## <a name="commit"></a> Git Commit Guidelines 
+
+### Commit Message Format
+A commit message is made up of a **header**, a **body** and a **footer**.  The header has a special
+format that includes a **type**, a **scope** and a **subject**:
+
+```
+<type>(<scope>): <subject>
+<BLANK LINE>
+<body>
+<BLANK LINE>
+<footer>
+```
+
+* The **header** and **scope** are mandatory
+* Lines shouldn't exceed 100 characters
+
+#### Type
+Must be one of the following:
+
+* **feat**: A new feature
+* **fix**: A bug fix
+* **refactor**: Refactoring change
+* **docs**: Documentation only changes
+* **style**: Style changes (white-space, formatting, missing semi-colons, etc)
+* **test**: Adding missing tests
+* **chore**: Changes to the build process or auxiliary tools and libraries
+
+#### Scope
+The scope sets the scope of the commit, for example `webapi`, `akka-app`, `doa`, etc...
+
+#### Subject
+Brief commit description
+
+* use the imperative, present tense: "change" not "changed" nor "changes"
+* don't capitalize first letter
```

Comment:
it's just an aesthetic thing, it keeps the whole commit message in a consistent case. And from my personal experience it makes the change log easier to read !